### ISSUE TITLE
Does this tool detect normal person fave too? I mean not bill gates or ronaldo 😄

### ISSUE THREAD

**Issue by mrhydra-np** [ageitgey/face_recognition#1235]

**Comment by KurtPrice** [comment]
Yes, but you must provide a set of images for the model to learn and associate with a name. This doesn't have a predefined model to recognize your face. For best results, take multiple photos at different angles and in different light conditions.

btw, github "issues" are not normally made for faqs.

**Comment by mrhydra-np** [comment]
Sorry for that 😢

**Comment by KurtPrice** [comment]
No worries. You'll just probably have a longer lead time to get your question answered. Most of these issues are going to be looked through for bugs or features to create so if someone is looking through them for that purpose they'll just ignore your question.
